applications/GeoMechanicsApplication/custom_workflows/adaptive_time_incrementor.cpp

```diff
@@ -85,6 +85,12 @@ void AdaptiveTimeIncrementor::PostTimeStepExecution(const TimeStepEndState& rRes
 
     // Avoid incrementing the time beyond the end time
     mDeltaTime = std::min(mDeltaTime, mEndTime - rResultantState.time);
+
+    // Avoid very small remaining time steps
+    double small_time_increment = 1.E-3 * mDeltaTime;
```

Comment:
I would prefer to make this variable `const` (since it is not supposed to be changed):
```suggestion
    const auto small_time_increment = 1.E-3 * mDeltaTime;
```
(And yes, I prefer `auto`, but I'll leave that up to you.)

Reply:
done